BTSPocket/Resources/Extensions/UIImage+extension.swift

```diff
@@ -23,12 +24,12 @@ extension UIImageView {
         }
     }
     
-    public func loadProfileImage(urlString: String) {
+    func loadProfileImage(urlString: String) {
         let url = URL(string: urlString)
         guard let requestURL = url else { fatalError("URl not valid") }
         var request: URLRequest = URLRequest(url: requestURL)
         request.httpMethod = "GET"
-        request.setValue(Constants.Endpoints.serverAddress, forHTTPHeaderField: "Referer")
+        request.setValue(Constants.serverAddress, forHTTPHeaderField: "Referer")
         let task = URLSession.shared.dataTask(with: request) { (DataResponse, URLResponse, Error) in
```

Comment:
If you use kingfisher, you can delete this

Reply:
just the functions? or the whole file? because the class will be empty when eliminating the functions that I don't need

Reply:
> just the functions? or the whole file? because the class will be empty when eliminating the functions that I don't need
If it's empty, complete file. If you have an extension w/ Kingfisher. Keep the file, remove unused function only